Makefile

```diff
@@ -171,10 +171,6 @@ $(LIBTAILSCALE): Makefile android/libs $(LIBTAILSCALE_SOURCES) $(GOBIN)/gomobile
 
 libtailscale: $(LIBTAILSCALE)
 
-tailscale-new-fdroid.apk: $(LIBTAILSCALE)
-	(cd android && ./gradlew test assembleFdroidDebug)
-	mv android/build/outputs/apk/fdroid/debug/android-fdroid-debug.apk $@
-
 tailscale-new-debug.apk: $(LIBTAILSCALE)
 	(cd android && ./gradlew test assemblePlayDebug)
```

Comment:
but this says "PlayDebug" .... is it using any Play Store / Google APIs?

Reply:
nope, thanks for pointing this out!